lib/api/1.1/northbound/profiles.js

```diff
@@ -2,9 +2,9 @@
 
 'use strict';
 
-var di = require('di'),
-    express = require('express'),
-    parser = require('body-parser');
+var di = require('di');
+var express = require('express');
+var ambiguousParser = require('./../../../fittings/ambiguous_text_parser');
```

Comment:
@benbp I'm ok with embedding it, if thats the direction we want to go with it. :)